cmd/build.go

```diff
@@ -274,12 +274,9 @@ func runBuild(args []string, mode mode, modeArgs []string) {
 		}
 	}
 
-	if mode == modeCoverage {
-		// Second pass with all targets
-		genInput.SelectedTargets = targets
-		genOutput = runGenerator(genInput)
-	}
-
+	// Second pass with all targets
+	genInput.SelectedTargets = targets
+	genOutput = runGenerator(genInput)
```

Comment:
I think this will run the generator now twice for all modes
Idk why it has to run twice for the coverage, but that is probably not what you want.
Instead set SelectedTargets already on the first run and keep the condition here?

Reply:
The problem is that apparently you need the first pass to enumerate all targets and then filter them...

I can have a look at how the targets are derived, maybe we can get them without actually running the generator